docs/theme/components/collocator/components/configuration-bar.tsx

```
@@ -0,0 +1,199 @@
+
+  const styles = useStyle();
+
+  const elementMap: Record<ComponentType, (d: IItem, parent?: string) => React.ReactElement> = {
+    [ComponentType.SWITCH]: (d: IItem, parent?: string) => <Switch name={d.name} onChange={(v) => {
+      if (!d.related) return
+
+      if (v) {
+        if (parent) {
+          const tempConfig = d.mergeRelated && isObject(d.mergeRelated) ? Object.keys(d.mergeRelated).reduce((pre, cur) => (
+            {
+              ...pre,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)